spring-cloud-open-service-broker-core/src/test/java/org/springframework/cloud/servicebroker/controller/CatalogControllerTest.java

```diff
@@ -42,11 +42,17 @@ void setUp() {
 	}
 
 	@Test
-	void catalogIsReturned() {
+	void catalogIsReturned() throws Exception {
 		Catalog expectedCatalog = Catalog.builder().build();
-		given(catalogService.getCatalog()).willReturn(Mono.just(expectedCatalog));
+
+		given(catalogService.getCatalog())
```

Comment:
Yeah, I agree with you. However, I'm not sure that test is even valuable. It may have been once upon a time, but the catalog integration tests do essentially the same thing and they also do a deep check on the catalog information. I was thinking last night what the difference in those tests should be, but everything comes back to mocks and being quite similar. I'll reconsider it for this PR though.